dictionaries/people-names/cspell-ext.json

```diff
@@ -0,0 +1,44 @@
+// cSpell Settings
+{
+    "id": "people-names",
+    "version": "0.2",
```

Comment:
Just want to confirm `0.2` is the right value

Reply:
Yes, it is the cSpell file format version.